### DISCUSSION TITLE
Testing Limits and Results

### DISCUSSION THREAD

**Discussion by nicholasjonnes** [zytzagoo/smtp-validate-email#65] [Q&A]
Hey,

I have a server setup and script ready to go to begin processing double opt-in list that I have for a client which is quite large (100k +). The list has not been engaged much in the past 18 months which is why I want to test everything. Does anyone have any rough guidelines for how many to process per day etc. to not annoy or bother email providers. I am not trying to do anything wrong, I just want to try and avoid getting blacklisted accidentally and then not be able to test my list or anything like that.

Also, for what is worth, if anyone is having connection issues when testing any email at all, there is a pretty high change that the fsockopen function is blocked by your hosting provider, or if you manage your own hardware it is not uncommon for it to be blocked with a default install as spammers have sadly abused this PHP function in the past.

**Comment by mmukhin** [comment]
@nicholasjonnes Did you ever do this? I'm curious what your results were like cleaning 100k emails.

**Reply by nicholasjonnes** [reply]
Sure did, best to start low. I think it was only doing 3k a day or something. Just have it in a cron that just ticks over every 10 minutes with small batches. Not worth flogging it or you will just be blocked by ISPs

Another tip: certain domains just are not worth trying to validate such as yahoo and bigpond.com. Easier to just mark as invalid and move on if you are working with volume
